vision/conv_mnist/Project.toml

```diff
@@ -1,14 +1,11 @@
 [deps]
 BSON = "fbb218c0-5317-5bc6-957e-2ee96dd4b1f0"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
-Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
```

Comment:
Of all the dependencies here, Flux is probably the most vital and shouldn't be removed ;)